test/Mono.Linker.Tests.Cases/LinkAttributes/LinkerAttributeRemoval.cs

```diff
@@ -15,13 +15,11 @@ namespace Mono.Linker.Tests.Cases.LinkAttributes
 	[SetupLinkAttributesFile ("LinkerAttributeRemoval.xml")]
 	[IgnoreLinkAttributes (false)]
 
-#if !ILLINK
-	[Reference ("System.dll")]
-#endif
 	[SetupCompileBefore ("attribute.dll", new[] { "Dependencies/LinkerAttributeRemovalAttributeToRemove.cs" })]
 	[SetupCompileBefore ("copyattribute.dll", new[] { "Dependencies/LinkerAttributeRemovalAttributeFromCopyAssembly.cs" })]
 	[SetupLinkerAction ("copy", "copyattribute")]
-#if !ILLINK
+#if !NETCOREAPP
+	[Reference ("System.dll")]
```

Comment:
is there still a case where this matters if we remove the mono build? illinker isn't used on Windows .NET Framework builds right?

Reply:
it's there only to make tests sharing easier with Unity